pemFioi/conceptDisplay-1.0.js

```diff
@@ -30,6 +39,38 @@ function conceptDisplay() {
       allLangDivs.show();
     }
   }
+
+  if (channel) {
+    var pythonCodes = $('.pythonCode');
+    pythonCodes.each(function (index, element) {
+      const jQueryElement = $(element);
+      if (!jQueryElement.parent().find('.pythonCode-execute').length) {
```

Comment:
This condition doesn't always work properly, and is probably unnecessary as you're already changing the class from pythonCode to pythonCode-container.

Reply:
Yes it's necessary because I'm only wrapping the .pythonCode element into a .pythonCode-container: the .pythonCode is still there and so I still need to check if it already has the container or not. I've changed the condition to something simpler that corresponds to what I'm saying. Would it be ok like that? And why wouldn't it work properly in the first case?